### PR TITLE
we can select any combination of repositories/organizations/open projects/closed projects based on the sidebar!

### PR DESCRIPTION
**Pull request by gladishd**

### CODE REVIEW

**Comment by aitikgupta** [comment]
I believe your filters are wrong, _but_ the UI seems reasonable.

File: webviews/store.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store';
+
+export const filters = writable(['Repository']);
```
Comment (aitikgupta) [inline webviews/store.js]: You're declaring a `filters` here as a writable, I don't see `get(filters)` anywhere?
Or does this line: https://github.com/Open-Sourced-Olaf/vscode-github-projects/blob/d5e13573e8ce756f8d52e778b1efa6b8555dffba/webviews/components/ProjectsList.svelte#L12

automatically gets the value from svelte store?

**Review by aitikgupta — APPROVED** [review]
Thanks for the UI @gladishd 😄 
Amazing work integrating everything @YuanSamuel 🎉

File: webviews/store.js (hunk 1 above)
Comment (YuanSamuel) [reply webviews/store.js]: I was playing with the store but couldn't get it working, let me remove that

Comment (aitikgupta) [reply webviews/store.js]: @YuanSamuel  ikr, Svelte store felt like a mess with us wanting svelte to re-render when there's a change in the store. :/

Comment (gladishd) [reply webviews/store.js]: He might have first tried cross-component store.js update and subscription, and instead he is sending `filters` as props from Home.svelte.  Can anyone confirm that store.js is not actually doing anything?  See?  @aitikgupta is truly a genius in questioning this assumption.

**Review by gladishd — COMMENTED** [review]
Okay, so store.js was indeed not doing anything also!  Are we actually getting all the repositories we need?

**Comment by gladishd** [comment]
> Okay, so store.js was indeed not doing anything also! Are we actually getting all the repositories we need?

I mean, are we still retrieving closed projects?

**Comment by aitikgupta** [comment]
Why did you close this @gladishd?

**Review by aitikgupta — APPROVED** [review]

**Pull request merged**